src/signTxOutput.c

```diff
@@ -70,7 +70,7 @@ static inline void advanceState()
 		ASSERT(subctx->currentAssetGroup < subctx->numAssetGroups);
 
 		// we are going to receive token amounts for this group
-		ASSERT(subctx->numTokens > 0);
+		VALIDATE(subctx->numTokens > 0, ERR_INVALID_DATA);
```

Comment:
The same as above, assertion is OK here.

Reply:
See above.